### PR TITLE
Make the quiet version of 'dotnet build' quieter.

### PR DESCRIPTION
**Pull request by rolfbjarne**
Ref: https://github.com/dotnet/sdk/issues/7986#issuecomment-604959822

### CODE REVIEW

**Review by mandel-macaque — APPROVED** [review]
🤫

**Review by dalexsoto — APPROVED** [review]
🤐

**Review by tj-devel709 — APPROVED** [review]

File: tests/Makefile [hunk 1]
```diff
@@ -213,6 +213,7 @@ $(TOP)/tools/common/SdkVersions.cs: $(TOP)/tools/common/SdkVersions.cs.in
 
 xharness/xharness.exe: $(xharness_dependencies) test.config test-system.config .stamp-src-project-files $(TOP)/tools/common/SdkVersions.cs
 	$(Q_GEN) $(SYSTEM_MSBUILD) /restore $(MSBUILD_VERBOSITY_QUIET) xharness/xharness.csproj
+xharness/xharness.csproj.inc: export BUILD_VERBOSITY=$(XBUILD_VERBOSITY)
 xharness/xharness.csproj.inc: export ABSOLUTE_PATHS=1
```
Comment (filipnavara) [inline tests/Makefile:217]: Looks like rule redefinition here. Doesn't GNU make ignore the first one?

Comment (rolfbjarne) [reply tests/Makefile:217]: This isn't actually a rule definition, it's a [target specific variable declaration](https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html).

It's setting the `BUILD_VERBOSITY` variable for _only_ the `xharness/xharness.csproj.inc` target.

**Comment by rolfbjarne** [comment]
Test failure is unrelated (https://github.com/xamarin/maccore/issues/2411).

File: tests/Makefile (hunk 1 above)
Comment (filipnavara) [reply tests/Makefile:217]: Thanks for explaining, learning something new every day...

Comment (mandel-macaque) [reply tests/Makefile:217]: ditto, and I though I new make.

**Pull request merged**